### PR TITLE
updated finished dfs graph

### PR DESCRIPTION
**Pull request by xialiu1988**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: Data-Structures/src/test/java/Graph/GraphTest.java [hunk 1]
```diff
@@ -94,4 +96,24 @@ public void bFSTraverse() {
...
+    {
+        Graph g = new Graph();
+        Vertex v1 = g.addNode("a");
+        Vertex v2 = g.addNode("b");
+        Vertex v3 = g.addNode("c");
+        g.addEdge(v1, v2, 3);
+        g.addEdge(v2, v3, 3);
+        g.addEdge(v1, v3, 3);
+        List<String> s = new ArrayList<>();
+        List<String> list = dFS(g.Vertices.get(0), s);
```
Comment (evanslaton) [inline Data-Structures/src/test/java/Graph/GraphTest.java:112]: I had to change this line to `g.dFS(g.Vertices.get(0), s)` in order to get it work
